portfolio/src/main/webapp/style.css

```diff
@@ -72,12 +72,39 @@ input[type=submit] {
   margin-bottom: 25px;
 }
 
+#text-container p {
+  font-size: 18px;
+}
+
 .comment {
   border-bottom: 1px solid white;
   margin-bottom: 10px;
   padding: 5px;
 }
 
+#number {
+  background-color: yellowgreen;
+  color: white;
+  border: none;
+  margin: 5px;
+  max-width: 100%;
+  padding: 5px 20px;
```

Comment:
I think 20px padding is too much - it makes the number disappear for me. The problem might be that it is in the middle of the menu but is hidden by the dropdown arrow.